backend/test/integration/job_api_test.go

```diff
@@ -207,12 +216,6 @@ func (s *JobApiTestSuite) TestJobApis() {
 	assert.Equal(t, 1, totalSize)
 	argParamsRun := runs[0]
 	s.checkArgParamsRun(t, argParamsRun, argParamsExperiment.ID, argParamsExperiment.Name, argParamsJob.ID, argParamsJob.Name)
-
```

Comment:
My understanding is that you make sure all the pipelines/runs/experiments/... are cleaned up before the test begins in SetupTests() methods, and therefore you don't need to clean up those pipelines/runs/experiments/... after the test ends, which enables the dev to debug integration test errors after the test ends. Is that correct? If so, can we add comments here as why we don't clean up after test ends (since conventionally we do)?

Reply:
Yes, that's correct.
Another benefit is, if you halt your test in the middle, rerunning the test won't fail because of cluster in an intermediate state.

Reply:
I just realized, frontend-integration-test actually fails after this.
So I guess either we need to add cluster cleanup in frontend-integration-test, or we still need clean up after each integration test.

Reply:
Updated implementation to use a flag to turn off resource clean up when developing it.